deploy/kubernetes/rbac.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
 rules:
   - apiGroups: [""]
     resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "update"]
+    verbs: ["get", "list", "watch", "update", "patch"]
```

Comment:
Do we still need update?

Reply:
I'd prefer to have it there, just in case :-)
(btw, I'd prefer if there were just generic "read" and "write" permissions)